app/models/concerns/fog_extensions/libvirt/server.rb

```diff
@@ -33,6 +33,18 @@ def vm_description
         _("%{cpus} CPUs and %{memory} memory") % {:cpus => cpus, :memory => number_to_human_size(memory.to_i)}
       end
 
+      # Other Fog CRs use .interfaces as the accessor, but libvirt does not
```

Comment:
Submit a change to the Fog provider in addition to this if you feel like Libvirt is not following conventions.